src/test/pythonnode_test.cpp

```diff
@@ -1026,3 +1026,42 @@ TEST_F(PythonFlowTest, ReloadWithDifferentScriptName) {
 
     checkDummyResponse("OUTPUT", data, req, res, 2 /* expect +2 */, 1, "mediaDummy");
 }
+
+TEST_F(PythonFlowTest, FailingToInitializeOneNodeDestructsAllResources) {
+    ConstructorEnabledModelManager manager;
+    std::string firstTestPbtxt = R"(
+    input_stream: "OVMS_PY_TENSOR:in"
+    output_stream: "OVMS_PY_TENSOR:out"
+        node {
+            name: "pythonNode1"
+            calculator: "PythonExecutorCalculator"
+            input_side_packet: "PYTHON_NODE_RESOURCES:py"
+            input_stream: "INPUT:in"
+            output_stream: "OUTPUT:inter"
+            node_options: {
+                [type.googleapis.com / mediapipe.PythonExecutorCalculatorOptions]: {
+                    handler_path: "/ovms/src/test/mediapipe/python/scripts/good_finalize_remove_file.py"
+                }
+            }
+        }
+        node {
+            name: "pythonNode2"
+            calculator: "PythonExecutorCalculator"
+            input_side_packet: "PYTHON_NODE_RESOURCES:py"
+            input_stream: "INPUT:inter"
+            output_stream: "OUTPUT:out"
+            node_options: {
+                [type.googleapis.com / mediapipe.PythonExecutorCalculatorOptions]: {
+                    handler_path: "/ovms/src/test/mediapipe/python/scripts/bad_initialize_return_false.py"
```

Comment:
Does this script simply return false? Could you use script that throws exception in initialize?
We don't want to return status in initialize, so this may get invalidated soon.

Reply:
done